modules/sortableBidAdapter.js

```diff
@@ -0,0 +1,143 @@
+    let loc = utils.getTopWindowLocation();
+
+    const sortableImps = utils._map(validBidReqs, bid => {
+      let rv = {
+        id: bid.bidId,
+        tagid: bid.params.tagId,
+        banner: {
+          format: utils._map(bid.sizes, ([width, height]) => ({w: width, h: height}))
+        },
+        ext: {}
+      };
+      if (bid.params.floor) {
+        rv.bidfloor = bid.params.floor;
+      }
+      if (bid.params.keywords) {
+        rv.ext.keywords = bid.params.keywords;
+      }
+      if (bid.params.bidderParams) {
+        utils._each(bid.params.bidderParams, (params, partner) => {
+          rv.ext[partner] = params;
+        });
+      }
+      if (bid.params.floorSizeMap) {
+        rv.ext.floorSizeMap = bid.params.floorSizeMap;
+      }
+      return rv;
+    });
+    const sortableBidReq = {
+      id: utils.getUniqueIdentifierStr(),
+      imp: sortableImps,
+      site: {
+        domain: loc.hostname,
+        page: loc.href,
+        ref: utils.getTopWindowReferrer(),
+        publisher: {
+          id: globalSiteId || validBidReqs[0].params.siteId,
+        },
+        device: {
+          w: screen.width,
+          h: screen.height
+        },
+      },
+    };
+    return {
+      method: 'POST',
+      url: `//${SERVER_URL}/openrtb2/auction?src=${REPO_AND_VERSION}&host=${loc.host}`,
+      data: JSON.stringify(sortableBidReq),
+      options: {contentType: 'text/plain'}
+    };
+  },
+
+  interpretResponse: function(serverResponse) {
+    const { body: {id, seatbid} } = serverResponse;
+    const sortableBids = [];
+    if (id && seatbid) {
+      utils._each(seatbid, seatbid => {
+        utils._each(seatbid.bid, bid => {
+          const bidObj = {
+            requestId: bid.impid,
+            cpm: parseFloat(bid.price),
+            width: parseInt(bid.w),
+            height: parseInt(bid.h),
+            creativeId: bid.crid || bid.id,
+            dealId: bid.dealid || null,
+            currency: 'USD',
+            netRevenue: true,
+            mediaType: BANNER,
+            ttl: 60
+          };
+          if (bid.adm && bid.nurl) {
+            bidObj.ad = bid.adm;
+            bidObj.ad += utils.createTrackPixelHtml(decodeURIComponent(bid.nurl));
+          } else if (bid.adm) {
+            bidObj.ad = bid.adm;
+          } else if (bid.nurl) {
+            bidObj.adUrl = bid.nurl;
+          }
+          sortableBids.push(bidObj);
+        });
+      });
+    }
+    return sortableBids;
+  },
+
+  getUserSyncs: (syncOptions, responses) => {
+    const sortableConfig = config.getConfig('sortable');
+    if (syncOptions.iframeEnabled && sortableConfig && !!sortableConfig.siteId) {
+      let syncUrl = `//${SERVER_URL}/sync?f=html&s=${sortableConfig.siteId}&u=${encodeURIComponent(utils.getTopWindowLocation())}`;
+      return [{
+        type: 'iframe',
+        url: syncUrl
+      }];
+    }
+  },
+
+  onTimeout(details) {
+    fetch(`//${SERVER_URL}/prebid/timeout`, {
```

Comment:
**fetch** isn't supported by Internet Explorer 10 or 11, and prebid supports IE 10+: https://caniuse.com/#search=fetch

The prebid **ajax** function should work as a substitute. If you search the adapters codebase, you'll find usage examples.